packages/paste-website/src/pages/inclusive-design/index.mdx

```diff
@@ -0,0 +1,409 @@
+        <ListItem>Mark invalid fields clearly and provide associated error messages</ListItem>
+        <ListItem>Do not auto focus form fields, on page load</ListItem>
+        <ListItem>Break up long and complex forms into discrete sections and/or screens</ListItem>
+        <ListItem>
+          Make forms as short as possible. For example, consider offering shortcuts like autocompleting an address by
+          using the postcode.
+        </ListItem>
+      </UnorderedList>
+    </DisclosureContent>
+  </Disclosure>
+  <Disclosure>
+    <DisclosureHeading as="h3" variant="heading30">
+      Text and typographic considerations
+    </DisclosureHeading>
+    <DisclosureContent>
+      <UnorderedList>
+        <ListItem>
+          Make sure heading levels describe a logical section/subsection structure{' '}
+          <Anchor showExternal href="https://webaim.org/techniques/semanticstructure/">
+            (Read more about this)
+          </Anchor>
+        </ListItem>
+        <ListItem>Translate / spell out acronyms the first time you use them</ListItem>
+        <ListItem>Employ well-balanced, highly legible fonts (not too complex or elaborate)</ListItem>
+        <ListItem>
+          Do not use very thin font faces{' '}
+          <Anchor
+            showExternal
+            href="http://www.telegraph.co.uk/science/2016/10/23/internet-is-becoming-unreadable-because-of-a-trend-towards-light/"
+          >
+            (Read an article about why this matters)
+          </Anchor>
+        </ListItem>
+        <ListItem>Underline links — at least in body copy</ListItem>
+        <ListItem>
+          Avoid justified body text since it creates rivers of spacing that can be hard to read.{' '}
+          <Anchor showExternal href="https://www.w3.org/TR/WCAG20-TECHS/F88.html">
+            (Read more about why this matters)
+          </Anchor>
+        </ListItem>
+        <ListItem>
+          Provide enough spacing between lines of text (`line-height`) to allow for readers to track text{' '}
+          <Anchor showExternal href="https://www.w3.org/TR/WCAG20-TECHS/C21.html">
+            (Read more about this)
+          </Anchor>
+        </ListItem>
+        <ListItem>
+          Avoid all-caps text{' '}
+          <Anchor showExternal href="https://github.com/humanmade/hm-pattern-library/issues/75">
+            (Read more about this)
+          </Anchor>
+        </ListItem>
+        <ListItem>
+          Ensure that content is written as clearly and simply as possible{' '}
+          <Anchor showExternal href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-supplements.html">
+            (Read more about why it matters)
+          </Anchor>
+        </ListItem>
+        <ListItem>
+          Warn users of links that have unusual behaviors, like linking off-site, or loading a new tab
+        </ListItem>
+        <ListItem>Use textual labels to make voice activation cues obvious</ListItem>
```

Comment:
what does a "textual label" mean here? is that as opposed to a label using an image / icon?

Reply:
Would "visible text labels" make more sense to you?